src/uct/sm/mm/base/mm_ep.c

```diff
@@ -563,14 +563,3 @@ ucs_status_t uct_mm_ep_flush(uct_ep_h tl_ep, unsigned flags,
     UCT_TL_EP_STAT_FLUSH(&ep->super);
     return UCS_OK;
 }
-
-ucs_status_t
-uct_mm_ep_check(uct_ep_h tl_ep, unsigned flags, uct_completion_t *comp)
-{
-    uct_mm_ep_t *ep = ucs_derived_of(tl_ep, uct_mm_ep_t);
-
-    UCT_EP_KEEPALIVE_CHECK_PARAM(flags, comp);
-    uct_ep_keepalive_check(tl_ep, &ep->keepalive, ep->fifo_ctl->pid, flags,
```

Comment:
I think we still may have an issue with calling `uct_ep_keepalive_init` inside mm endpoint initialization.
E.g. when PID namespace is not shared between two containers running on the same host.